install/upgrade_to_2.14.php

```diff
@@ -59,6 +59,17 @@ public function addEntityOption() {
             'condition' => 'WHERE `entities_id` = 0'
          ]
       );
+
+      $this->migration->addField(
+         $table,
+         'is_category_visible',
+         'integer', [
+            'after' => 'home_page',
+            'value' => '1',
```

Comment:
Maybe it is  better to inherit from parent entity ?

```suggestion
            'value' => '-2',
```